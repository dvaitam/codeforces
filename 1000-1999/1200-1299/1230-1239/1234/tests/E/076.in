14 6
13 10 4 5 5 9
