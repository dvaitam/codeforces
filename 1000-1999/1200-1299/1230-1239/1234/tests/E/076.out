14 14 14 20 20 12 12 12 14 21 13 13 21 14
